### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by bafain**

### CODE REVIEW

File: content/first-order-logic/completeness/completeness-thm.tex [hunk 1]
```diff
@@ -42,15 +42,15 @@
 \Gamma$, $\iftag{FOL}{\Sat{M(\Gamma^*)}}{\pSat{v(\Gamma^*)}}{!A}$, so
 $\Gamma$ is satisfiable.\iftag{FOL}{
   If $\Gamma$ does contain~$\eq$,
-  then by \olref[ide]{lem:truth}, $\Sat{M/_\approx}{!A}$ iff $!A \in
+  then by \olref[ide]{lem:truth}, $\Sat{M/\mathord\approx}{!A}$ iff $!A \in
```
Comment (rzach) [inline content/first-order-logic/completeness/completeness-thm.tex]: Should actually be `\equivclass{M}{\approx}` -- that makes the typesetting of equivalence classes uniform.

File: content/first-order-logic/completeness/completeness-thm.tex [hunk 2]
```diff
@@ -42,15 +42,15 @@
...
   \Gamma^*$ for all !!{sentence}s~$!A$.  In particular,
-  $\Sat{M/_\approx}{!A}$ for all $!A \in \Gamma$, so $\Gamma$ is
+  $\Sat{M/\mathord\approx}{!A}$ for all $!A \in \Gamma$, so $\Gamma$ is
```
Comment (rzach) [inline content/first-order-logic/completeness/completeness-thm.tex]: ditto

**Comment by rzach** [comment]
Thanks for all of these. Any more coming?

**Comment by bafain** [comment]
These are the last two. Thanks for the rapid response!

**Comment by rzach** [comment]
Thanks again!

**Pull request merged**